src/renderers/WebGLRenderer.js

```diff
@@ -1005,6 +1005,69 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 				if ( geometryAttribute !== undefined ) {
 
+					var dataType = _gl.FLOAT;
+					var normalized = false;
+					if( geometryAttribute.array instanceof Float32Array ) {
```

Comment:
BTW How about
```diff
+ var array = geometryAttribute.array;
```
and then use `array` instead of `geometryAttribute.array` all over the place?

Not does only does it avoid code that can't minify, it also helps analysers (as in optimizing JIT compilers) to see that there can be no side-effects. Consider the following code

```javascript
obj.foo.bar();
func();
obj.foo.baz();
```
now in order to perform sub expression elimination of `obj.foo`, the compiler must be 100% sure, that `func` doesn't change `obj.foo`. That's not always so easy to figure out, especially not in a short time, and much easier with local variables.